packages/fxa-payments-server/server/config/index.js

```diff
@@ -2,7 +2,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-'use strict';
+
```

Comment:
Was this line breaking something?

Reply:
Not that I can think of. I think I touched this file while debugging, and the linter auto deleted this. Should I put it back?

Reply:
Nah.  Just curious on the change.